recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.10.8" %}
+{% set version = "1.10.9" %}
 
 package:
   name: holoviews
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/h/holoviews/holoviews-{{ version }}.tar.gz
-  sha256: 2ce380ed1f678efadbeb3c376e2b90f7bd67e489db6f308c2d5abefabc37b36a
+  url: https://assets.holoviews.org/packages/holoviews-1.10.9.tar.gz
```

Comment:
If the releases will always be published in the non-standard URL from now on, by non-standard I mean not PyPI or GitHub, we can at least use the `jinja` variable to avoid confusing the bot.

Reply:
This URL should be the exception and not the rule. The next release should go back to the PyPI URL.